src/runtime/dom-particle.ts

```diff
@@ -104,7 +112,7 @@ export class DomParticle extends XenStateMixin(DomParticleBase) {
   }
 
   /** @deprecated */
-  get _views() {
```

Comment:
sync your client, this was just removed.

Reply:
I'll talk to you about this. I can't get the change (just pulled from `upstream`).

Reply:
was missed before. Resolved in discussion. 
Meant: `setViews`.